packages/new-polymath-ui/src/components/inputs/Checkbox/Checkbox.tsx

```diff
@@ -49,6 +49,7 @@ export const CheckboxPrimitive: FC<Props> = ({
           onChange={e => {
             onChange(e.target.checked);
           }}
+          style={{ border: 'red' }}
```

Comment:
Not sure why there is a red border applied here.

Reply:
This was a mistake. Will be removed